gsa/src/web/pages/reports/detailspage.js

```diff
@@ -153,6 +159,20 @@ class ReportDetails extends React.Component {
     }
   }
 
+  startMeasurement() {
```

Comment:
Actually it could measure all kind of durations  but currently it's only used for the load duration. So I suppose you are right with changing the name would be better.

Reply:
As it can measure any kind of duration something like startDurationMeasurement() would be quite informative. We don't need to restrict this method to startLoadDurationMeasurement() or anything like that.